doc/user/content/manage/user-management/manage-roles.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Manage roles"
+description: "Create and manage roles in Materialize"
+menu:
+  main:
+    parent: user-management
+    weight: 15
+---
+
+{{< alpha />}}
+
+This page outlines how to create and manage roles in Materialize.
+
+## Create a role
+
+To create a new role, use the `CREATE ROLE` statement:
+
+```sql
+CREATE ROLE <role_name>;
+```
+
```

Comment:
I think you need to spell out the options, like you've done for `ALTER ROLE`. All of the options seem to be documented here: https://materialize.com/docs/sql/create-role/#syntax.

Reply:
I've moved the attribute information in the Create section and moved Alter up so both sections are closer together and highlight the attributes that can be created and modified